package.json

```
@@ -7,7 +7,7 @@
     "packages/wallet-service"
   ],
   "engines": {
-    "node": ">=18"
+    "node": ">=20"
```

chore: We're using v22 now
```suggestion
    "node": ">=22"
```

---

Done